tests/src/common/WskCli.java

```diff
@@ -982,9 +989,9 @@ public RunResult cli(boolean verbose, int expectedExitCode, String... params) th
     public RunResult cli(boolean verbose, int expectedExitCode, File workingDir, String... params) throws IllegalArgumentException, IOException {
         String[] cmd;
         if (usePythonCLI) {
-            cmd = WhiskProperties.useCliDownload() ? new String[] { getDownloadedCliPath() } : new String[] { WhiskProperties.python, new File(cliPath).toString() };
+            cmd = WhiskProperties.useCLIDownload() ? new String[] { WhiskProperties.python, getDownloadedCLIPath() } : new String[] { WhiskProperties.python, new File(cliPath).toString() };
```

Comment:
Not sure what the point creating a new file then printing the string value of the path is here: new File(cliPath).toString().